### ISSUE TITLE
how to exact the color in .xef?

### ISSUE THREAD

**Issue by cjhyy7190** [LuciaXu/Xef2Mat-Jpg#2]
when I run the .exe ,I found it can't exact the color .How can I do that?thank you

**Comment by ekurtgl** [comment]
Yes, color doesn't seem to be extracted. Can you help us out @LuciaXu ?

**Comment by cjhyy7190** [comment]
最近找到一个不防错的扑克站，春节辨没事就过来消遣吧纠，我今年春节的红包就靠它了，哈哈，http://www.sqp6h1.cn
